pkg/informant/consts.go

```diff
@@ -16,7 +16,7 @@ const (
 	AgentBackgroundCheckTimeout time.Duration = 250 * time.Millisecond
 
 	AgentResumeTimeout  time.Duration = 100 * time.Millisecond
-	AgentSuspendTimeout time.Duration = 200 * time.Millisecond
+	AgentSuspendTimeout time.Duration = 5 * time.Second        // may take a while; it /suspend intentionally waits
```

Comment:
why 5 seconds? 

[I read this code a few times before I realized what's happening is you make a timeout longer (because a thing takes longer in practice than we initially thought) and then fiddle with the locks so that you (maybe?) get a method to block until another operation returns. It might be clearer (and actually reflect your intent) to have there be a wait-group-ish barrier rather than a mutex barrier for the operation in question. Having said that, this is probably fine.]